packages/ripple-binary-codec/test/shamap.test.ts

```diff
@@ -72,18 +80,18 @@ describe('ShaMap', () => {
             hashPrefix() {
               return leafNodePrefix
             },
-            toBytesSink(sink) {
+            toBytesSink(sink: BytesList) {
               sink.put(bytes)
             },
-          }
+          } as ShaMapLeaf
         })
-        .forEach((so) => map.addItem(so.index, so))
+        .forEach((so: ShaMapLeaf) => map.addItem(so.index, so))
       expect(map.hash().toHex()).toBe(ledger.account_hash)
       // console.log('took seconds: ', (now() - t));
     })
   }
-  factory('ledger-full-38129.json')
-  factory('ledger-full-40000.json')
+  factory(ledgerFull38129)
+  factory(ledgerFull40000)
   // factory('ledger-4320277.json');
   // factory('14280680.json');
```

Comment:
Should we get rid of these comments or add these tests? (Do these files still exist?)

Reply:
Good call.  These are gone now.